### PR TITLE
added comments on mv and > potentially silently overwriting existing files

### PR DESCRIPTION
**Pull request by jevbelikov**

### CODE REVIEW

**Comment by gdevenyi** [comment]
This commit needs to be rebased against the newest version of the repo, it seems to be pulling in changes @ChristinaLK has made to the repo.

**Comment by ChristinaLK** [comment]
I was planning to defer this until after the lesson release.

**Comment by gvwilson** [comment]
Yup, please defer.

File: 02-create.md [hunk 1]
```diff
@@ -248,6 +248,11 @@ $ ls thesis
 quotes.txt
 ~~~
 
+One has to be careful when specifying target file name, since `mv` will
```
Comment (ChristinaLK) [inline 02-create.md]: "specifying *the* target file name"

File: 03-pipefilter.md [hunk 2]
```diff
@@ -107,11 +107,12 @@ $ wc -l *.pdb > lengths.txt
...
 The greater than symbol, `>`, tells the shell to **redirect** the command's output
-to a file instead of printing it to the screen.
-The shell will create the file if it doesn't exist,
-or overwrite the contents of that file if it does.
-(This is why there is no screen output:
-everything that `wc` would have printed has gone into the file `lengths.txt` instead.)
+to a file instead of printing it to the screen. The shell will create 
+the file if it doesn't exist. In case the file exists, it will be 
+silently overwritten, which may lead to data loss and thus requires
+some caution. (This is why there is no screen output:
```
Comment (ChristinaLK) [inline 03-pipefilter.md]: I might move this parenthetical note to after "...printing it to the screen." in line 110.

File: 03-pipefilter.md [hunk 3]
```diff
@@ -107,11 +107,12 @@ $ wc -l *.pdb > lengths.txt
 ~~~
 
 The greater than symbol, `>`, tells the shell to **redirect** the command's output
-to a file instead of printing it to the screen.
-The shell will create the file if it doesn't exist,
-or overwrite the contents of that file if it does.
-(This is why there is no screen output:
-everything that `wc` would have printed has gone into the file `lengths.txt` instead.)
+to a file instead of printing it to the screen. The shell will create 
+the file if it doesn't exist. In case the file exists, it will be
```
Comment (ChristinaLK) [inline 03-pipefilter.md]: "In case the" --> "If the" - shorter and equally clear

**Comment by ChristinaLK** [comment]
Sorry for returning to this so late @jevbelikov !  Let me know what you think about the suggested line changes (and check where the merge conflicts are - I'm guessing in the 06-find.md changes), and I'll try to get this merged in asap.

**Comment by jevbelikov** [comment]
I agree with the changes, also apologies for the need of rebasing. Probably I should have create a branch for the change and then request a PR from the branch?

**Comment by ChristinaLK** [comment]
That is our typical workflow - if you want to create a branch for these changes and submit a new pull request, that's totally fine!

**Comment by ChristinaLK** [comment]
Hi @jevbelikov , do you want to make these changes, or have me add them myself (I'll use your initial commits, so it should be credited to you)?  Either way is fine, just let me know!

**Comment by jevbelikov** [comment]
@ChristinaLK it would be great if you could add the change this time around, thanks.

**Pull request merged**